features/Insights_Advisor_affected_clusters_version.feature

```diff
@@ -0,0 +1,218 @@
+            | Bug12345 | 10 days ago | Important  | high       | high   |
+        And the user USER1 is already logged in into Hybrid Cloud Console
+        When user looks at Hybrid Cloud Console main page
+        Then menu on the left side should be displayed
+        And the left menu might contain these top level items
+            | Left menu item           | Required for this test |
+            | Overview                 | no                     |
+            | Releases                 | no                     |
+            | Downloads                | no                     |
+            | Advisor                  | yes                    |
+            | Subscriptions            | no                     |
+            | Cost Management          | no                     |
+            | Cluster Manager Feedback | no                     |
+            | Red Hat Marketplace      | no                     |
+            | Documentation            | no                     |
+        When user expand "Advisor" top level item
+        Then the menu should be expanded under "Advisor" top level item
+        And following new items should be displayed in the sub-menu on the left side
+            | Expanded menu item | Required for this test |
+            | Clusters           | no                     |
+            | Recommendations    | yes                    |
+        When user select "Recommendations" menu item from this sub-menu
+        Then an "Advisor recommendations" page should be displayed right of the left menu bar
+        And widget with filter settings should be displayed
+        And table with several columns should be displayed
+            | Column name |
+            | Name        |
+            | Added       |
+            | Total risk  |
+            | Clusters    |
+        And that table should contain at least one row
+            | Column name | Value       |
+            | Name        | Bug1234     |
+            | Added       | 10 days ago |
+            | Total risk  | Important   |
+            | Clusters    | 1           |
+        When user clicks on an "Bug1234" link
+        Then new page with additional information about selected recommendation should be displayed
+        And the following values needs to be displayed
+            | Value type  | Content             | Displayed as              |
+            | Description | Textual description | Text                      |
+            | KB article  | Link to KB article  | Link                      |
+            | Total risk  | Important           | Widget (icon+label)       |
+            | Likelihood  | High                | Widget (thermometer-like) |
+            | Impact      | High                | Widget (thermometer-like) |
+        And "Affected clusters" table needs to be displayed below additional info
+            | Name                                 | Clickable (link) | Version |
```

Comment:
You skipped the "Last seen" column (not sure if it is necessary for your BDD test, but worth mentioning still)

Reply:
That's intentional because for this test case I just need to check for the version. There are others that check for many other fields, so I tried to keep this simple :)